src/app/pages/Discover/Discover.module.css

```diff
@@ -0,0 +1,30 @@
+::-webkit-scrollbar {
+  display: none;
+}
+
+.container {
+  height: 100vh;
+  display: grid;
+  background-color: var(--bg-primary);
+  grid-template-rows: 20% 70% 10%;
+  grid-auto-flow: row;
+}
+
+.topSection {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  padding: 2.5rem 0 0 1rem;
+  align-items: center;
+}
+
+.cardWrapper {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
```

Comment:
Let's have this be responsive. It should be `1fr` on iPhone 5 and then scale up to have two or more rows if it fits